src/main.py

```diff
@@ -130,9 +131,14 @@ def commit_changes(changes):
 )
 
 
-def open_pr(subject, body, branch):
+def open_pr(subject, body, branch, manifest_checker=None):
+    try:
+        github_token = os.environ["GITHUB_TOKEN"]
+    except KeyError:
+        log.error("GITHUB_TOKEN environment variable is not set")
+        return
```

Comment:
If this is an error, the script should not exit(0).

Reply:
Done.